sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -6379,7 +6379,7 @@ class AstBuilder extends DataTypeAstBuilder
    *
    * For example:
    * {{{
-   *   DECLARE [OR REPLACE] [VARIABLE] [db_name.]variable_name
+   *   DECLARE [OR REPLACE] [VARIABLE] [db_name.]variable_name [COMMA [db_name.]variable_name]*
```

Comment:
```suggestion
   *   DECLARE [OR REPLACE] [VARIABLE] variable_name [COMMA variable_name]*
```
A qualified variable name can have many name parts. Let's just use `variable_name`.